packages/backend-common/src/s3.ts

```diff
@@ -58,7 +58,8 @@ export const getSignedDownloadUrl = async (
 		{ expiresIn }, // override default expiration time of 15 minutes
 	);
 
-export const getFile = async (
+// for larger files (such as media files)- stream the file to disk
+export const streamObjectToFile = async (
```

Comment:
This is not used anywhere right now, are you thinking it'll be used in the future?

Reply:
it is used in packages/backend-common/src/s3.ts

Reply:
Yea it is used in `getFileFromS3` function but getFileFromS3 doesn't seem to be used anywhere

Reply:
sorry you're absolutely right, I had assumed this must be used when we donwload the media files but that's been replaced by `getObjectWithPresignedUrl`

Reply:
Removed in https://github.com/guardian/transcription-service/pull/48/commits/2389593f125beca55762b079f60c05e02e346499